cilium/Dockerfile

```diff
@@ -1,20 +1,20 @@
-ARG GOLANG_IMAGE=quay.io/cybozu/golang:1.18-focal
-ARG UBUNTU_IMAGE=quay.io/cybozu/ubuntu:20.04
+ARG GOLANG_IMAGE=quay.io/cybozu/golang:1.20-jammy
+ARG UBUNTU_IMAGE=quay.io/cybozu/ubuntu:22.04
 ARG DESTDIR=/tmp/install/linux/amd64
 
 # Stage1: build common
 FROM ${GOLANG_IMAGE} as build-base
 ARG DESTDIR
 ENV DESTDIR=${DESTDIR}
 COPY TAG /
-COPY 18449.patch /tmp/
+COPY cilium.patch /tmp/
```

Comment:
I think we can clone our fork repository, including the patches, instead of patching the original repository
`git clone --depth 1 --branch <BRANCH> https://github.com/cybozu-go/cilium`